README.md

```diff
@@ -30,7 +30,8 @@ Add the following annotations to your podSpec to mount the secret in your pod
 
   ```secrets.k8s.aws/secret-arn: <SECRET-ARN>```
 
-The decrypted secret is written to a volume named `secret-vol` and the filename of the secret is the name within secrets manager. The Kubernetes dynamic admission controller also creates corresponding mountPath `/tmp/secrets` for containers within the pod to access the secret.
+
+The decrypted secret is written to a volume named `secret-vol`.  <SECRET-ARN> can also include the mount path, like: <SECRET-ARN>:<CONTAINER-MOUNT-PATH> for each secret. If the mount path is not specified, then the secret is mounted in `/tmp/<secret-name>`. It supports multiple secrets comma-concatenated `<SECRET1-ARN>,<SECRET2-ARN>`. The Kubernetes dynamic admission controller creates corresponding mountPath and subPath for all containers within the pod to access the secret.
 
```

Comment:
Since we can't push the container images to Dockerhub, can you include directions in the README that explain how to run the `makefile`, e.g. `make --repository [name]` or how to build/install the admission controller.

Reply:
good call, just updated the README.md. Do you think it is simple enough for anyone to use?